### PR TITLE
Enable some custom modifier tests on Mono

### PR DESCRIPTION
**Pull request by lambdageek**
After mono/mono@c0b48793db422b502364f5836ac9ba8abe37aa7a, Mono should be able to run these tests.

/cc @jaredpar

### CODE REVIEW

File: src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadInAttributeModifier.cs [hunk 1]
```diff
@@ -313,7 +313,6 @@ class Test
                 Diagnostic(ErrorCode.ERR_BindToBogus, "M").WithArguments("RefTest.M(in int)").WithLocation(4, 38));
         }
 
-        [ConditionalFact(typeof(ClrOnly))] // https://github.com/mono/mono/issues/6936
```
Comment (jaredpar) [inline src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadInAttributeModifier.cs]: This should be `[Fact]` now. Some for rest of changes in this file.

File: src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadingMethods.cs [hunk 2]
```diff
@@ -1310,7 +1310,6 @@ static void Main()
         }
 
         [WorkItem(666162, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/666162")]
-        [ClrOnlyFact(ClrOnlyReason.Ilasm)]
```
Comment (jaredpar) [inline src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadingMethods.cs]: This should be [Fact] now. Some for rest of changes in this file.

File: src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs [hunk 3]
```diff
@@ -1107,7 +1107,6 @@ public void MixedRefType()
             //Assert.True(comp.References[1].EmbedInteropTypes);
         }
 
-        [Fact]
```
Comment (jaredpar) [inline src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs]: Why did you delete this?

File: build/Targets/RepoToolset/RepositoryInfo.targets [hunk 4]
```diff
@@ -8,10 +8,12 @@
   <!--
     Include both GitHub and VSTS packages to enable SourceLink in repositories that mirror to VSTS.
   -->
+  <!-- DO NOT MERGE: enable linux build
```
Comment (jaredpar) [inline build/Targets/RepoToolset/RepositoryInfo.targets]: Why was this necessary? Is the sourcelink breaking for you and if so what was the error?

**Comment by lambdageek** [comment]
Put back the `[Fact]`s and rebased on top of roslyn master to try and get a clean diff.

@jaredpar how do I run just one test suite now?  on the old branch I was doing
```
./build/scripts/tests.sh Debug mono Microsoft.CodeAnalysis.CSharp.Symbol
```

but that doesn't exist anymore.

**Comment by lambdageek** [comment]
It's failing.  As expected (I guess).  Looks like it's downloading Mono 6.1.0.188, which is too old.
Need to wait for a newer mono and re-trigger the job.

**Comment by lambdageek** [comment]
It ran with mono 6.1.0.545 and passed 🎉

**Comment by jaredpar** [comment]
Thanks for getting these turned back on!
